tests/observability/network/conftest.py

```diff
@@ -191,3 +194,24 @@ def invalid_cnao_operator(prometheus, admin_client, hco_namespace, csv_scope_cla
 
     linux_bridge_plugin_ds = DaemonSet(name=KUBE_CNI_LINUX_BRIDGE_PLUGIN, namespace=hco_namespace.name)
     linux_bridge_plugin_ds.wait_until_deployed(timeout=TIMEOUT_10MIN)
+
+
+@pytest.fixture(scope="class")
+def csv_image_updated_with_bad_image(invalid_cnao_operator):
+    def get_csv_image():
```

Comment:
no need for an additional function, can be implemented in the sampler block
 lambda: invalid_cnao_operator.instance.spec.install.spec.deployments ....